src/screens/OrganizationVenues/OrganizationVenues.tsx

```
@@ -133,13 +144,21 @@
+    console.error(
+      'GraphQL Error Response:',
+      JSON.stringify(venueError, null, 2),
+    );
     errorHandler(t, venueError);
   }
 
   // Updating venues state when venue data changes
   useEffect(() => {
-    if (venueData && venueData.getVenueByOrgId) {
-      setVenues(venueData.getVenueByOrgId);
+    if (venueData && venueData.organization) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)